cvs/tests/rccl/rccl_heatmap_cvs.py

```
@@ -267,7 +264,7 @@ def test_disable_firewall( phdl ):
 
 def pytest_generate_tests(metafunc):
     config_file = metafunc.config.getoption("config_file")
-    if not config_file:
+    if not config_file or not os.path.exists(config_file):
         return
```

Nitpick: This is a silent return, would be nice we can throw a error or print something, hard to debug when this code path is executed